content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -352,12 +352,28 @@ For example:
         Content-Type: application/json
         Accept: application/json
 
-The response would look the same as for non dry run request, but the values of some generated fields may differ.
+The response would look the same as for non-dry-run request, but the values of some generated fields may differ.
 
+### Dry-run authorization
+
+Authorization for dry-run and non-dry-run requests is identitical. Thus, to make
```

Comment:
I spotted another typo:
```suggestion
Authorization for dry-run and non-dry-run requests is identical. Thus, to make
```
(sorry I didn't see this before)

Reply:
Thank you!!